docs/providers/fn/cli-reference/deploy.md

```diff
@@ -0,0 +1,32 @@
+<!--
+title: Serverless Framework Commands - Fn - Deploy
+menuText: deploy
+menuOrder: 2
+description: Deploy your service to the specified provider
+layout: Doc
+-->
+
+<!-- DOCS-SITE-LINK:START automatically generated  -->
+### [Read this on the main serverless docs site](https://www.serverless.com/framework/docs/providers/fn/cli-reference/deploy)
+<!-- DOCS-SITE-LINK:END -->
+
+# Fn - Deploy
+
+The `sls deploy` command deploys your entire service via the Fn API. Run this command when you have made service changes (i.e., you edited `serverless.yml`).
+
+Use `serverless deploy function -f my-function` when you have made code changes and you want to quickly upload your updated code to your Fn server or cluster.
+
+```bash
+serverless deploy
+```
+
+This is the simplest deployment usage possible. With this command Serverless will deploy your service to the configured Fn server.
+
+## Options
+- `--noDeploy` or `-n` Skips the deployment steps and leaves artifacts in the `.serverless` directory.
```

Comment:
In general, `--noDeploy` option has been deprecated in other providers, so it would be good to implement it as `serverless package` command for consistancy.

Reply:
I missed removing this. There is no --noDeploy flag. There is also no Package command.